Source: John Kallimanis
Upstream: Not applied by upstream. Created for porting reasons.
Reason: Strncpy function is provided by glibc.
diff -crB ephem/main.c ephem-4.30/main.c
*** ephem/main.c	2013-09-22 17:26:33.465170091 +0300
--- ephem-4.30/main.c	2013-09-22 16:45:02.311313800 +0300
***************
*** 23,29 ****
--- 23,32 ----
  #include "circum.h"
  #include "screen.h"
  
+ /* On GNU systems strncpy is internal.*/
+ /*
  extern char *strncpy();
+ */
  extern char *getenv();
  
  /* shorthands for fields of a Now structure, now.
